### PR TITLE
ARM64 Support for External DNS

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*

ARM64 Support for External DNS

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/addons/external-dns/index.ts [hunk 1]
```diff
@@ -22,9 +22,9 @@ const defaultProps = {
     name: 'external-dns',
     chart: 'external-dns',
     namespace: 'external-dns',
-    repository: 'https://charts.bitnami.com/bitnami',
+    repository: 'https://kubernetes-sigs.github.io/external-dns/',
     release: 'blueprints-addon-external-dns',
-    version: '6.20.3',
+    version: '1.12.0',
```
Comment (youngjeong46) [inline lib/addons/external-dns/index.ts:27]: fyi the latest version is at 1.13. Not sure if that one's ARM compatible.

**Review by youngjeong46 — COMMENTED** [review]
Left one comment. Otherwise LGTM.

File: lib/addons/external-dns/index.ts (hunk 1 above)
Comment (shapirov103) [inline lib/addons/external-dns/index.ts:27]: is there a warning on deploy when using 1.12? There should be. 
Has it been confirmed to work with arm?

**Review by shapirov103 — COMMENTED** [review]
LGTM , however let's move to the latest version if possible. I will kick off e2e in the mean time

File: lib/addons/external-dns/index.ts (hunk 1 above)
Comment (elamaran11) [reply lib/addons/external-dns/index.ts]: ya nice catch, fixed. It works now with `1.13.0`

Comment (elamaran11) [reply lib/addons/external-dns/index.ts]: @shapirov103 It did throw a warning which i ignored. I fixed it now and it works perfectly fine with ARM. Please run e2e and merge.

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**